squad/frontend/templatetags/squad.py

```diff
@@ -140,6 +140,11 @@ def build_section_url(build, name):
     return reverse(name, args=(build.project.group.slug, build.project.slug, build.version))
 
 
+@register_global_function
+def download_build_attachments_url(group_slug, project_slug, build_version, testrun, filename, name):
+    return reverse(name, args=(group_slug, project_slug, build_version, testrun, filename))
```

Comment:
Is it used with different name than 'build_attachments' ?

Reply:
I don't get what you mean here @mwasilew

Reply:
why do you pass 'name' variable when it's always the same?

Reply:
changed this one @mwasilew